app/error-handling-controllers.js

```diff
@@ -1,8 +1,20 @@
-function errorHandler400s(error, res, req, next) {
-  return res.status(404).send({ msg: "404 error path not found" });
+function errorHandler400s(error, req, res, next) {
+  if (error.msg === "couldn't find article") {
```

Comment:
We could also make this more dynamic by just checking error.msg and error.status exist then send those back :) just an idea for refactoring later